src/main/java/frc/robot/RobotMap.java

```diff
@@ -5,5 +5,8 @@ public class RobotMap {
     private RobotMap() {}
 
     // add constants here
-    // public static final type NAME = value;
+    public static final int PISTON_FORWARD_CHANNEL = 1;
+    public static final int PISTON_BACKWARD_CHANNEL = 2;
```

Comment:
```suggestion
    public static final int CORAL_ARM_PISTON1_BACKWARD_CHANNEL = 1;
```
Name must be specific